source/init/sentry.ts

```diff
@@ -7,7 +7,17 @@ function install() {
 		return
 	}
 
-	Sentry.init({dsn: SENTRY_DSN})
+	Sentry.init({
+		dsn: SENTRY_DSN,
+
+		tracesSampleRate: 0.2,
+
+		integrations: [
+			new Sentry.ReactNativeTracing({
+				tracingOrigins: ['localhost', 'my-site-url.com', /^\//u],
```

Comment:
Do we need the my-site-url line?

Reply:
Oops, you are correct. I expect that we should change that to frogpond.tech…